app/controllers/hq/partners_controller.rb

```diff
@@ -21,14 +20,13 @@ def show
   end
 
   def edit
-    load_partner
-    load_associations
+    @facade = PartnerFacade.new(load_partner)
   end
 
   def update
     load_partner
     build_partner
-    save_partner or re_render 'edit'
+    save_partner or render 'edit'
```

Comment:
perhaps load the facade in `load_partner` or have a `load_facade` method to keep things dry, as a failed save in update will require a handle to the `@facade` instance variable